tools/do-destroy-leaked-templates

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -e
+
+export DJANGO_SETTINGS_MODULE=zproject.test_settings
+
+for template in "$@"
+do
+    psql --quiet -v ON_ERROR_STOP=1 -h localhost postgres zulip_test << EOF
```

Comment:
Do you think there is a better way of doing this where we go to postgres shell just once with all the databases that need to be dropped?

Reply:
I'll look into this further, and test out different approaches here.

Reply:
@adnrs96 with your suggestion via chat, I'm seeing a nice return on time to when the number of templates increases.  With 20 leaked templates I'm seeing:

Previous implementation:
```
real	0m3.631s
user	0m0.592s
sys	0m0.167s
```
Down to:
```
real	0m2.610s
user	0m0.039s
sys	0m0.005s
```

(I'm interested in digging a little deeper here, but pushed you're suggested changes since it's certainly an improvement.)